Geolocation service class (#296)
infinum/eightshift-libs: src/Geolocation/Readme.md

# GeoLocation data

> We use [GeoIP2-php library](https://github.com/maxmind/GeoIP2-php) to get the correct countries based on the IP address of the user that we compare with the countries list that we get from the [DataHub list](https://datahub.io/core/country-list).

Reviewer: ```suggestion
We use the [GeoIP2-php library](https://github.com/maxmind/GeoIP2-php) to get the user's location (country) based on their IP address. List of the countries is compared with the [DataHub list](https://datahub.io/core/country-list).
```
